easybuild/easyconfigs/l/libICE/libICE-1.0.9-GCC-11.3.0.eb

```
@@ -0,0 +1,29 @@
+easyblock = 'ConfigureMake'
+
+name = 'libICE'
+version = '1.0.9'
+
+homepage = "http://www.freedesktop.org/wiki/Software/xlibs"
+description = """X Inter-Client Exchange library for freedesktop.org"""
+
+toolchain = {'name': 'GCC', 'version': '11.3.0'}
```

Since `libICE` should be used a dependency `plotutils` which should be used a dependency for https://github.com/easybuilders/easybuild-easyconfigs/pull/15950 I think it should also be in the `GCCcore` toolchain
```suggestion
toolchain = {'name': 'GCCcore', 'version': '11.3.0'}
```